src/prefect/types/__init__.py

```diff
@@ -18,7 +18,6 @@
 
 class Duration(timedelta):
     schema: ClassVar = core_schema.timedelta_schema(
-        ge=timedelta(seconds=0),
```

Comment:
This was an oversight from my last PR, we don't want any restrict the base `Duration` field